src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/TwitterWidgetPageObject.java

```diff
@@ -11,13 +13,17 @@
 
   @FindBy(css = ".widget-twitter iframe")
   private WebElement twitterIframe;
+  @FindBy(css = ".widget-twitter iframe")
+  private List<WebElement> twitterIframeList;
   @FindBy(css = "div.timeline")
   private WebElement twitterBody;
 
   private static final String TAG_NAME = "twitter";
   private static final String ARTICLE_NAME = "TwitterWidget";
-  private static final String TAG =
-      "<twitter widget-id=\"522824386202447873\" screen-name=\"sfbart\" />";
+  private static final String TAGS[] = {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move the array designator from the variable to the type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1197)